pkg/compact/compact.go

```diff
@@ -402,6 +402,10 @@ func (cg *Group) Key() string {
 	return cg.key
 }
 
+func (cg *Group) Metadata() []*metadata.Meta {
+	return cg.metasByMinTime
+}
```

Comment:
I guess this is not needed, right? We don't need to have a method to return all metadata of this group. We only need a method for Group to remove some metadatas.

Reply:
This is used by the `Plan` method, not for removing metadata. I'm creating a separate method for that.

Reply:
Then why do we need this method in `Plan`? We can just return `metasByMinTime` directly in `Plan`, right? It is the same package